ons_alpha/core/blocks/related.py

```diff
@@ -18,11 +18,16 @@
 class LinkBlockStructValue(StructValue):
     @cached_property
     def link(self) -> dict | None:  # pylint: disable=inconsistent-return-statements
+        title, desc = self.get("title"), self.get("description")
```

Comment:
Nit: This is clearer split onto multiple lines
```suggestion
        title = self.get("title")
        desc = self.get("description")
```

Reply:
Done